cabal.project

```diff
@@ -38,6 +38,12 @@ package *
   ghc-options: -haddock
   test-show-details: direct
 
+source-repository-package
+    type: git
+    location: https://github.com/kokobd/lsp.git
```

Comment:
minor thing, we could use lsp master, as your [pr ](https://github.com/haskell/lsp/pull/388) was merged into

Reply:
Yeah, and we should do a release before this is done.